Dockerfile

```diff
@@ -1,4 +1,8 @@
 FROM centos:7
+
+ARG USER_ID=14
+ARG GROUP_ID=50
```

Comment:
Any specific reason for using 14 and 50 as uid and gid?

Reply:
@fauria Sure. This is the default values of ftp:ftp
You can try to install fresh container and check it.
```
# id -u ftp
14
# id -g ftp
50
```